authhelper.go

```diff
@@ -119,8 +119,8 @@ func main2(stdout io.Writer, config configData) {
 
 type Claims struct {
 	jwt.RegisteredClaims
-	Email      string `json:"email"`
-	DBUsername string `json:"dbUsername"`
+	Email    string `json:"email"`
+	Username string `json:"username"`
 }
 
 func (c Claims) Valid() error {
```

Comment:
Should we check that either username or sub is set here?

Reply:
And return the "Missing x in claims" error as we do for email

Reply:
Hmm, that valid check wasn't being invoked.
And yes, I think you're right.  We should require 'username' or 'sub'. 
Doing that and fixing tests.